components/dashboard/DashboardWeeklySummary.tsx

```
@@ -1,35 +1,37 @@
-      </div>
-      
-      <div className="space-y-4">
-        <div>
-          <div className="flex justify-between items-center mb-1">
-            <span className="text-sm text-gray-600">Task Completed</span>
-            <span className="text-2xl font-bold">14</span>
-          </div>
+      <div className="rounded-lg border bg-white p-4 shadow-xs">
+        <div className="mb-4 flex items-center justify-between">
+          <h3 className="text-lg font-semibold">Weekly Summary</h3>
+          <button className="cursor-pointer text-sm text-blue-600 hover:text-blue-700">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)